### ISSUE TITLE
Unhandled Exception when using OrdinaryLeastSquaresRegressor

### ISSUE THREAD

**Issue by helloguo** [dotnet/machinelearning#329]
### System information

- **OS version/distro Windows 10**:
- **.NET Version 2.1.300-rc1-008673**: 

### Issue

- **What did you do?**
I replace `FastTreeRegressor` with `OrdinaryLeastSquaresRegressor` in sample Regression_TaxiFarePrediction https://github.com/dotnet/machinelearning-samples/blob/master/samples/getting-started/Regression_TaxiFarePrediction/Program.cs#L65
- **What happened?**
error msg:
```
=============== Training model ===============
Automatically adding a MinMax normalization transform, use 'norm=Warn' or 'norm=No' to turn this behavior off.
  Bad value at line 1 in column PassengerCount
  Bad value at line 1 in column TripDistance
Processed 100001 rows with 2 bad values and 0 format errors
  Bad value at line 1 in column PassengerCount
  Bad value at line 1 in column TripDistance
  Bad value at line 1 in column FareAmount
Processed 100001 rows with 3 bad values and 0 format errors
Trainer solving for 19 parameters across 100000 examples

Unhandled Exception: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.NotSupportedException: The MKL library (Microsoft.ML.MklImports.dll) or one of its dependencies is missing.
   at Microsoft.ML.Runtime.Learners.OlsLinearRegressionTrainer.TrainCore(IChannel ch, Factory cursorFactory, Int32 featureCount)
   at Microsoft.ML.Runtime.Learners.OlsLinearRegressionTrainer.Train(RoleMappedData examples)
   --- End of inner exception stack trace ---
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Object[] arguments, Signature sig, Boolean constructor)
   at System.Reflection.RuntimeMethodInfo.UnsafeInvokeInternal(Object obj, Object[] parameters, Object[] arguments)
   at Microsoft.ML.Runtime.Data.TrainUtils.TrainCore(IHostEnvironment env, IChannel ch, RoleMappedData data, ITrainer trainer, String name, RoleMappedData validData, ICalibratorTrainer calibrator, Int32 maxCalibrationExamples, Nullable`1 cacheData, IPredictor inpPredictor)
   at Microsoft.ML.Runtime.EntryPoints.LearnerEntryPointsUtils.Train[TArg,TOut](IHost host, TArg input, Func`1 createTrainer, Func`1 getLabel, Func`1 getWeight, Func`1 getGroup, Func`1 getName, Func`1 getCustom, ICalibratorTrainerFactory calibrator, Int32 maxCalibrationExamples)
   at Microsoft.ML.Runtime.Learners.OlsLinearRegressionTrainer.TrainRegression(IHostEnvironment env, Arguments input)

```
- **What did you expect?**
I was wondering how to use `OrdinaryLeastSquaresRegressor`?
### Source code / logs

Please paste or attach the code or logs or traces that would be helpful to diagnose the issue you are reporting.

**Comment by shauheen** [comment]
Thank you @helloguo for filing this issue. I think the issue has been previously reported (#234 and #241). @Ivanidzo4ka is working on a temporary fix in #286 . Please let me know if this can be closed.

**Comment by helloguo** [comment]
Closing it because of duplicating other issues.
